WordPress/Classes/Networking/BlogServiceRemoteXMLRPC.m

```diff
@@ -104,66 +101,62 @@ - (void)updateBlogSettings:(RemoteBlogSettings *)remoteBlogSettings
 
 
 
-- (WPXMLRPCRequestOperation *)operationForOptionsWithSuccess:(OptionsHandler)success
-                                                    failure:(void (^)(NSError *error))failure
+- (void)operationForOptionsWithSuccess:(OptionsHandler)success
+                               failure:(void (^)(NSError *error))failure
```

Comment:
I agree, just wanted to make sure that the first step of this changes was more linear, instead of moving all the code around. I can do this on this PR if you think it's ok.